lib/stdlib/src/rand.erl

```diff
@@ -2232,7 +2232,7 @@ The low bits of the base generator are surprisingly good, so the lowest
 weaknesses that lie in the high bits of the 32-bit MWC "digit".
 It is recommended to use `rem` on the the generator state, or bit mask
 extracting the lowest bits to produce numbers in a range 16 bits or less.
-See the recepies at the start of this
+See the recipies at the start of this
```

Comment:
I am no spelling bee but according my understanding should it not say: recipes!?

Reply:
Oh my gosh, I was so fixated on the first `i` that I didn't notice the second part that was wrong! One sec... I'll fix this. So sorry!

Reply:
Fixed. Sorry about that!

Reply:
No problem, I will leave it for the original author to merge on Monday when all test has run (should really just be a formality for this kind of change, but anyway).